bitbucket_hg_exporter/__main__.py

```diff
@@ -764,14 +764,14 @@ def thread_fn(i, message_queue, credentials):
                     clone_dest = os.path.join(self.__settings['project_path'], 'git-repos', *github_data['name'].split('/'))
                     clone_url =  github_data['repository']['clone_url']
                     if not os.path.exists(os.path.join(clone_dest, '.git', 'config')):
-                        self.call_git_subprocess('git', 'clone', clone_url, clone_dest, error_message='Failed to git clone {}'.format(clone_url))
+                        self.call_git_subprocess('clone', clone_url, cwd=clone_dest, error_message='Failed to git clone {}'.format(clone_url))
```

Comment:
```suggestion
                        self.call_git_subprocess('clone', clone_url, clone_dest, error_message='Failed to git clone {}'.format(clone_url))
```

I think the `cwd` kwarg needs to be reverted here otherwise it would clone into the path ...../git-repos/owner/repo_name/repo_name if my understanding of the git clone manual page is correct. `clone_dest` should be an argument to `git clone` (not the cwd) otherwise git clone uses the human part of the clone URL to make a new directory.

Reply:
thanks, I actually ran into that exact problem while testing this; I've applied your suggestion